src/ert/_c_wrappers/enkf/enkf_fs_manager.py

```diff
@@ -8,6 +10,11 @@
     from ert._c_wrappers.enkf import EnsembleConfig
     from ert._clib.state_map import StateMap
 
+FS_VERSION = 0
```

Comment:
Does this mean that the `current` file system version is `0`?

Reply:
yes